DXMainClient/DXGUI/Multiplayer/ChatListBox.cs

```diff
@@ -52,17 +52,22 @@ public void AddMessage(string sender, string message, Color color)
 
         public void AddMessage(ChatMessage message)
         {
+            var listBoxItem = new XNAListBoxItem
+            {
+                TextColor = message.Color,
+                Selectable = true,
+                Tag = message
+            };
+            
             if (message.SenderName == null)
-                AddItem(Renderer.GetSafeString(string.Format("[{0}] {1}",
+                listBoxItem.Text = Renderer.GetSafeString(string.Format("[{0}] {1}",
                     message.DateTime.ToShortTimeString(),
-                    message.Message), FontIndex),
-                    message.Color, true);
+                    message.Message), FontIndex);
             else
-            {
-                AddItem(Renderer.GetSafeString(string.Format("[{0}] {1}: {2}",
-                    message.DateTime.ToShortTimeString(), message.SenderName, message.Message), FontIndex),
-                    message.Color, true);
-            }
+                listBoxItem.Text = Renderer.GetSafeString(string.Format("[{0}] {1}: {2}",
+                    message.DateTime.ToShortTimeString(), message.SenderName, message.Message), FontIndex);
```

Comment:
Any multi-line code blocks should use braces for better readability and excluding the possibility of [goto fail error](https://www.synopsys.com/blogs/software-security/understanding-apple-goto-fail-vulnerability-2/).

Reply:
Fixed.